packages/mds-db/migration.ts

```diff
@@ -130,11 +134,24 @@ async function doMigration(exec: SqlExecuterFunction, migration: MIGRATION, migr
   }
 }
 
+async function alterAuditEventsColumns(exec: SqlExecuterFunction) {
+  await exec(
+    `ALTER TABLE ${schema.TABLE.audit_events} ADD COLUMN ${schema.COLUMN.provider_event_id} ${schema.COLUMN_TYPE.provider_event_id}`
+  )
+  await exec(
+    `ALTER TABLE ${schema.TABLE.audit_events} ADD COLUMN ${schema.COLUMN.provider_event_type} ${schema.COLUMN_TYPE.provider_event_type}`
+  )
+  await exec(
+    `ALTER TABLE ${schema.TABLE.audit_events} ADD COLUMN ${schema.COLUMN.provider_event_type_reason} ${schema.COLUMN_TYPE.provider_event_type_reason}`
+  )
+}
+
 async function doMigrations(client: MDSPostgresClient) {
   const exec = SqlExecuter(client)
   // All migrations go here. createMigrationsTable will never actually run here as it is inserted when the
   // migrations table is created, but it is included as it provides a template for how to invoke them.
   await doMigration(exec, 'createMigrationsTable', async () => {})
+  await doMigration(exec, 'alterAuditEventsColumns', alterAuditEventsColumns)
```

Comment:
This becomes ` await doMigration(exec, 'alterAuditEventsColumns', alterAuditEventsColumnsMigration)`